.chloggen/TEMPLATE.yaml

```diff
@@ -1,7 +1,7 @@
 # One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
 change_type:
 
-# The name of the component, or a single word describing the area of concern, (e.g. operator, target allocator, github action)
+# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opapm, github action)
```

Comment:
```suggestion
# The name of the component, or a single word describing the area of concern, (e.g. collector, target allocator, auto-instrumentation, opamp bridge, github action)
```

Reply:
If the label is just `opamp`, then I think it should be changed as well. WDYT @jaronoff97 ?

Reply:
I have changed the label to `opamp`

Reply:
In that case, you have a typo there.

Reply:
oh, fixed